metrics_utility/management/commands/gather_automation_controller_billing_data.py

```diff
@@ -108,13 +125,7 @@ def _handle_ship_target(self, ship_target):
     def _handle_datelike(self, value, help=''):
         if not value:
             return None
-
-        if value.isdigit():
-            raise UnparsableParameter(f'Bare numbers are not valid ({help})')
```

Comment:
This check was necessary, people were calling it with bare numbers and getting weird behaviour, because `parser.parse(2000)` just gives you a `datetime` that's today at midnight, but in the year 2000, no timezone, while `parser.parse(20)` gives you the 20th of the current month, etc.

Reply:
So before we get to this function we validate the values of of since and until to ensure they match.  That validation begins [here](https://github.com/ansible/metrics-utility/pull/128/files/61fd7301933dd18b996a71f2796d291ba8f1006a#diff-bb98f7ff0b0f4e4c66ebfeb4296c00cf73d8e86eafcb1fd12ba5c7eca4aeb2a7R85).  Bare numbers are no longer allow and will throw a validation error.

Reply:
Validation there is done by calling `parse_date_param`.

But that one does accept bare numbers, precisely because it ends up in the `parser.parse` branch, which accepts those.